test/gov/init_exchange.sh

```diff
@@ -207,6 +207,26 @@ echo "Registering Anax device1 in customer org..."
 REGANAX1C=$(curl -sLX PUT $CERT_VAR --header 'Content-Type: application/json' --header 'Accept: application/json' -u "Customer1/icpadmin:icpadminpw" -d '{"token":"abcdefg","name":"anaxdev","registeredServices":[],"msgEndPoint":"","softwareVersions":{},"publicKey":"","pattern":"","arch":"amd64"}' "${EXCH_URL}/orgs/Customer1/nodes/an12345" | jq -r '.msg')
 echo "$REGANAX1C"
 
+DEVICE_NUM=6
+if [ ${MUL_AGENTS} -lt 5 ]; then
+  NUM_AGENTS=$((${MUL_AGENTS}+$DEVICE_NUM))
+else
+  NUM_AGENTS=$((4+$DEVICE_NUM))
+fi
+while [ ${DEVICE_NUM} -lt ${NUM_AGENTS} ]; do
+  echo "Registering Anax device${DEVICE_NUM}..."
```

Comment:
Do we have to register the external agents this way? is there no way we could use hzn register for them?

Reply:
Defer to future issue